content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,393 @@
+
+   {{< text bash >}}
+   $ kubectl exec -n spire spire-server-0 -- \
+   /opt/spire/bin/spire-server entry create \
+   -spiffeID spiffe://example.org/ns/default/sa/httpbin \
+   -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+   -selector k8s:ns:default \
+   -selector k8s:pod-uid:$HTTPBIN_POD_UID \
+   -dns $HTTPBIN_POD \
+   -socketPath /run/spire/sockets/server.sock
+   {{< /text >}}
+
+{{< warning >}}
+SPIFFE IDs for workloads must follow the Istio SPIFFE ID pattern: `spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>`
+
+{{< /warning >}}
+
+See the [SPIRE help on Registering workloads](https://spiffe.io/docs/latest/deploying/registering/) to learn how to create new entries for workloads and get them attested using multiple selectors to strengthen attestation criteria.
+
+## Verifying that identities were created for workloads
+
+Confirm that identities were created for the workloads.
+   {{< text bash >}}
+   $ kubectl exec -i -t $SPIRE_SERVER_POD -n spire -c spire-server -- /bin/sh -c "bin/spire-server entry show -socketPath /run/spire/sockets/server.sock"
+   Found 3 entries
+   Entry ID         : c8dfccdc-9762-4762-80d3-5434e5388ae7
+   SPIFFE ID        : spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:istio-system
+   Selector         : k8s:pod-uid:88b71387-4641-4d9c-9a89-989c88f7509d
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+
+   Entry ID         : af7b53dc-4cc9-40d3-aaeb-08abbddd8e54
+   SPIFFE ID        : spiffe://example.org/ns/default/sa/httpbin
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:default
+   Selector         : k8s:pod-uid:ee490447-e502-46bd-8532-5a746b0871d6
+   DNS name         : httpbin-5f4d47c948-njvpk
+
+   Entry ID         : f0544fd7-1945-4bd1-88dc-0a5513fdae1c
+   SPIFFE ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Parent ID        : spiffe://example.org/spire/server
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s_psat:agent_ns:spire
+   Selector         : k8s_psat:agent_sa:spire-agent
+   Selector         : k8s_psat:cluster:demo-cluster
+   {{< /text >}}
+
+* Check istio-ingressgateway pod state:
+
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   1/1     Running   0          60s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          45s
+{{< /text >}}
+
+After registering an entry for istio-ingressgateway pod, Envoy receives the identity issued by SPIRE and uses it for all TLS and mTLS 
+communications.
+
+### Check Certificate

```

Comment:
After this heading, could you write a sentence about what this section is about like you did for the previous section? Maybe something like: "Use the following commands to verify that the identity issued to the sample httpbin service deployed in the manual registration example above is an X.509 certificate issued by SPIRE."

I'm assuming that this section only applies to the manual registration method you outlined earlier? 

I'd like to make this heading a little more specific. Does this make sense?
```suggestion
### Check that the httpbin identity was issued by SPIRE
```